src/game_renderer/InputBindings.cpp

```diff
@@ -7,6 +7,9 @@
 static ImGuiIO prev_imgui_io;
 static ImGuiIO new_imgui_io;
 
+int button_count; 
+static const unsigned char* new_buttons;
```

Comment:
Since ImGui doesn't support gamepad inputs, we will have to make some kind of data structure here to hold onto the joystick inputs rather than just keep the pointer returned by glfwGetJoystickButtons